server/planning/assignments/assignments.py

```diff
@@ -862,7 +863,9 @@ def _get_empty_updates_for_assignment(self, assignment):
 
     def _set_user_for_assignment(self, assignment, assignee, assignor=None):
         updates = self._get_empty_updates_for_assignment(assignment)
-        updates["assigned_to"]["user"] = assignee
+
+        if not get_config_assignment_manual_reassignment_only():
+            updates["assigned_to"]["user"] = assignee
 
         if assignor:
```

Comment:
If the config is turned off, then we should probably not change the `assignor_user` either (as no re-assignment was performed)

Reply:
got it